src/script/sign.h

```diff
@@ -81,4 +91,15 @@ void UpdateInput(CTxIn& input, const SignatureData& data);
  * Solvability is unrelated to whether we consider this output to be ours. */
 bool IsSolvable(const SigningProvider& provider, const CScript& script);
 
+class SignatureExtractorChecker : public BaseSignatureChecker
+{
+private:
+    SignatureData* sigdata;
```

Comment:
Commit "Make SignatureData able to store signatures and scripts"

Use references instead of pointers?

Reply:
Done